modules/installation-network-user-infra.adoc

```diff
@@ -193,7 +193,7 @@ endif::ibm-z-kvm[]
 
 .4+|TCP
 |`1936`
-|Metrics
+|Metrics. Can receive internal and external HTTPS traffic.
```

Comment:
```suggestion
|Metrics. Can receive the internal and external HTTPS traffic.
```

Reply:
This suggestion would be wrong, in the sense that this is not the only port which receives HTTPS traffic (it is not even the main one), so saying that it may receive **the** internal and external HTTPS traffic, can lead to the false impression that it is the one and only HTTPS port and be extremely misleading.

Reply:
Hi @palonsoro . Thanks for the comment. Would stating "both internal and external HTTPS traffic" work for you?

Reply:
Thanks, @palonsoro . I've removed this sentence from the table entry.